src/pages/Embeds/Plio.vue

```diff
@@ -0,0 +1,667 @@
+<template>
```

Comment:
Can't make it out from here if there has been some significant change to the code and where it has been